app/views/projects/show.html.erb

```diff
@@ -2,32 +2,32 @@
 <%- share_url project_url(@project); %>
 
 <div class="container">
-  <div class="bg-white px-4 py-5 border-b border-gray-200 sm:px-6">
-    <div class="-ml-4 -mt-4 flex justify-between items-center flex-wrap">
-      <div class="ml-4 mt-4">
-        <h3 class="text-lg leading-6 font-medium text-gray-900">
-          <%= @project.name %> project
-        </h3>
+  <% if current_user == @project.user %>
+    <div class="bg-white px-4 py-5 border-b border-gray-200 sm:px-6">
+      <div class="-ml-4 -mt-4 flex justify-between items-center flex-wrap">
+        <div class="ml-4 mt-4">
+          <h3 class="text-lg leading-6 font-medium text-gray-900">
+            <%= @project.name %> project
```

Comment:
Should we really add "project" at the end?

Reply:
Yeah, I was wondering that too. Left it in because it was in the original page. But, agree with you, it's probably not needed/should be removed.